app/views/vm_common/_resize.html.haml

```diff
@@ -18,7 +19,7 @@
       .col-md-8
         %select{:name        => 'flavor_id',
                 'ng-model'   => 'vm.vmCloudModel.flavor_id',
-                'ng-options' => 'flavor.id as flavor.name for flavor in vm.flavors track by flavor.id',
+                'ng-options' => 'flavor.id as flavor.name for flavor in vm.flavors',
                 :miqrequired => true,
                 :checkchange => true}
```

Comment:
`checkchange` should not be used with `form-changed`, you should be able to remove each checkchange here now :)

Reply:
ok...